addons/common/functions/fnc_getVehicleIcon.sqf

```diff
@@ -35,7 +35,11 @@ if (isNil "_cachedValue") then {
         if (_vehicleValue != "" && {((toLowerANSI _vehicleValue) find ".paa") > -1}) then {
             _cachedValue = _vehicleValue;
         } else {
-            _cachedValue = DEFAULT_TEXTURE;
+            if (_vehicleValue select [0,1] == "\" && {fileExists (_vehicleValue + ".paa")}) then {
```

Comment:
Why `_vehicleValue select [0,1] == "\"`?

Reply:
Given that this runs only once and the result is cached, imo it's not a problem.

Reply:
Agreed.